test/e2e/e2e_test.go

```diff
@@ -182,4 +182,50 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 		})
 	})
+
+	Context("Creating a GKE cluster without autopilot", func() {
+		It("Should create a cluster with 1 machine pool and scale", func() {
+			By("Initializes with 1 machine pool")
+
+			clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+				ClusterProxy: bootstrapClusterProxy,
+				ConfigCluster: clusterctl.ConfigClusterInput{
+					LogFolder:                clusterctlLogFolder,
+					ClusterctlConfigPath:     clusterctlConfigPath,
+					KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+					InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+					Flavor:                   "ci-gke",
+					Namespace:                namespace.Name,
+					ClusterName:              clusterName,
+					KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
+					ControlPlaneMachineCount: pointer.Int64Ptr(1),
+					WorkerMachineCount:       pointer.Int64Ptr(1),
+				},
+				WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
+				WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
+				WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+				WaitForMachinePools:          e2eConfig.GetIntervals(specName, "wait-worker-machine-pools"),
+			}, result)
+
+			// By("Scaling worker node to 3")
+			// clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+			// 	ClusterProxy: bootstrapClusterProxy,
+			// 	ConfigCluster: clusterctl.ConfigClusterInput{
+			// 		LogFolder:                clusterctlLogFolder,
+			// 		ClusterctlConfigPath:     clusterctlConfigPath,
+			// 		KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+			// 		InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+			// 		Flavor:                   clusterctl.DefaultFlavor,
+			// 		Namespace:                namespace.Name,
+			// 		ClusterName:              clusterName,
+			// 		KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
+			// 		ControlPlaneMachineCount: pointer.Int64Ptr(1),
+			// 		WorkerMachineCount:       pointer.Int64Ptr(3),
+			// 	},
+			// 	WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
+			// 	WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
+			// 	WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+			// }, result)
+		})
+	})
```

Comment:
is this duplicated (as we have test in test/e2e/e2e_gke_test.go)?

Reply:
Ah, i moved this to a different file and forgot. Thanks for this.